src/test/skript/tests/syntaxes/effects/EffGlowingText.sk

```diff
@@ -0,0 +1,16 @@
+test "glowing sign blocks":
+	set {_loc} to spawn of (random world out of all worlds)
```

Comment:
I think this is better \:)
```suggestion
	set {_loc} to spawn of "world"
```

Reply:
the only thing about this is it relies on the world being named "world". I think this is probably a safe bet, but that was my logic here. what do you think?

Reply:
i think it's okay since the default world (at least for tests) will always be called world :)